src/Microsoft.TemplateEngine.Abstractions/ITemplateInfo.cs

```diff
@@ -49,6 +49,7 @@ public interface ITemplateInfo
 
         IReadOnlyDictionary<string, IBaselineInfo> BaselineInfo { get; }
 
+        [Obsolete("This property is obsolete")]
```

Comment:
Nit: dot in the end.

Reply:
should we have analyzer rule on it? :)

Reply:
I like how Vlada thinks... I agree, nits like this should be covered via analyzers or not mentioned in PRs :P